hooks/validator/README.md

```diff
@@ -0,0 +1,86 @@
+# Validator Hook
+
+The `validator hook` taps into the `After` lifecycle event to validate the result of flag evaluations.
+
+The hook defines a `validator` interface with function
+```go
+IsValid(flagEvaluationDetails of.EvaluationDetails) error
+```
+to allow application authors to supply their own validators.
+
+There are, however, [ready to be used validators](#validators) that conform to the interface.
```

Comment:
An optional function with ready-to-use validators is a great way to go, nice work.